### ISSUE TITLE
About the dataset PhotoMatte85

### ISSUE THREAD

**Issue by Datura0822** [PeterL1n/BackgroundMattingV2#143]
Hi, I want to know how the dataset PhotoMatte85 is evaluated quantitatively, because the dataset PhotoMatte85 does not contain trimap. I download the dataset PhotoMatte85 from https://grail.cs.washington.edu/projects/background-matting-v2/#/datasets.

**Comment by Datura0822** [comment]
If you use trimap in your evaluation, can you share it with me?

**Comment by PeterL1n** [comment]
None of the datasets contain gt trimap. They are created through dilation and erosion of the alpha. The detail parameter I believe is specified in the paper, likely in supplementary.

**Comment by Datura0822** [comment]
OK, I got it. The details are described as follows

The trimaps used as input for trimap-based methods and for defining the error metric regions are obtained by thresholding the grouth-truth alpha between 0.06 and 0.96, then applying 10 iterations of dilation followed by 10 iterations of erosion using a 3×3 circular kernel.

But i don't know how to code “by thresholding the grouth-truth alpha between 0.06 and 0.96”, is it possible to share the code in the paper?

It’s very good that you have replied to me, it’s too much trouble for you.

**Comment by PeterL1n** [comment]
No, the code is not kept. It is pretty straightforward. Look into cv2's dilation and erosion methods.
